.travis.yml

```diff
@@ -1 +1,11 @@
 language: java
+
+dist: trusty
+
+jdk:
+  - oraclejdk8
```

Comment:
Pretty sure you could use openjdk8 rather than Oracle, as the rest of our stuff seems to be using that now.